rwgame/RWGame.cpp

```diff
@@ -212,31 +216,40 @@ void RWGame::handleCheatInput(char symbol) {
 #endif
         checkForCheat(armor_cheat, [&] {
             player->getCurrentState().armour = 100.f;
-            // @todo ShowHelpMessage("CHEAT4"); // III / VC: Inputting armor
-            // cheat.
+            showHelpMessage("CHEAT4"); // III / VC: Inputting armor cheat.
         });
 
         checkForCheat("GUNSGUNSGUNS", [&] {
-            // @todo give player weapons
-            // @todo ShowHelpMessage("CHEAT2"); // III / VC: Inputting weapon
-            // cheats.
+            const std::vector<int> ammo = {1,  //baseball bat
+                                           100,//pistol
+                                           100,//uzi
+                                           20, //shotgun
+                                           5,  //grenade
+                                           5,  //molotov
+                                           5,  //rocket launcher
+                                           20, //flamethrower
+                                           200,//ak47
+                                           200,//m16
+                                           5   //sniper rifle
+                                           };
+            for (unsigned long i = 0; i < ammo.size(); i++)
```

Comment:
better to use ```std::size_t```

Reply:
or even ```size_type```

Reply:
Thank you

Reply:
It seems like that using `size_type` will cause warning about comparison of integer expressions of different signedness. So I'll think it better to use something like `std::vector<size_type>::size_type`

Reply:
Yeah, that's what I meant, it should be  ```std::vector<int>::size_type```